__docs__/wonder-blocks-accordion/accordion.stories.tsx

```diff
@@ -684,4 +694,177 @@ const styles = StyleSheet.create({
+                    >
+                        <Button
+                            onClick={() =>
+                                setToggleModeForNextedSection("expand-all")
+                            }
+                            disabled={
+                                toggleModeForNextedSection === "expand-all"
+                            }
+                        >
+                            Expand All
+                        </Button>
+                        <Button
+                            onClick={() =>
+                                setToggleModeForNextedSection("collapse-all")
+                            }
+                            disabled={
+                                toggleModeForNextedSection === "collapse-all"
+                            }
+                        >
+                            Collapse All
+                        </Button>
+                    </View>
+
+                    {/* Main Accordion Component */}
+                    <Accordion
+                        toggleMode={toggleModeForNextedSection}
+                        onToggleModeComplete={({
+                            anyExpanded,
+                            anyCollapsed,
+                        }: AccordionToggleModeStatus) => {
+                            // partial
+                            if (anyExpanded && anyCollapsed) {
+                                setToggleModeForNextedSection("none");
+                            } else if (!anyExpanded) {
+                                // all closed
+                                setToggleModeForNextedSection("collapse-all");
+                            } else {
+                                // all open
+                                setToggleModeForNextedSection("expand-all");
+                            }
+                        }}
+                    >
+                        <AccordionSection
+                            header="Section 1"
+                            caretPosition="start"
+                        >
+                            This is the content of section 1.
+                        </AccordionSection>
+                        <AccordionSection
+                            header="Section 2"
+                            caretPosition="start"
+                        >
+                            This is the content of section 2.
+                        </AccordionSection>
+                        <AccordionSection
+                            header="Section 3"
+                            caretPosition="start"
+                        >
+                            This is the content of section 3.
+                        </AccordionSection>
+                        <AccordionSection
+                            header={`Nested Section 1`}
+                            key={`nested-section-1`}
+                            caretPosition="start"
+                        >
+                            <AccordionSection
```

Comment:
Can you add a small margin around the inner nested sections so it's visually more clear that they're nested?